src/wallet.ts

```diff
@@ -0,0 +1,2170 @@
+        tags: true,
+        content: 'Payment ID: (optional)'
+    });
+
+    // defind password textbox
+    let idInput = blessed.textbox({
+        parent: transferForm,
+        name: 'paymentid',
+        top: 5,
+        left: 0,
+        width: 40,
+        mouse: true,
+        vi: false,
+        inputOnFocus: true,
+        height: 3,
+        border: {
+            type: 'line',
+            fg: 'white'
+        },
+        fg: 'white',
+    });
+
+    idInput.key('enter', function() {
+        transferButton.press();
+    });
+
+    idInput.key(['escape'], function() {
+        screen.focusPop();
+        transferWindow.focus();
+    })
+
+    idInput.on('blur', function() {
+        transferWindow.focus();
+    })
+
+    // pop focus on textbox click
+    idInput.on('click', function() {
+        screen.focusPop();
+    });
+
+    // define password textbox label
+    let amountLabel = blessed.text({
+        parent: transferForm,
+        top: 8,
+        left: 0,
+        fg: 'white',
+        content: 'Amount:'
+    });
+
+    // defind password textbox
+    let amountInput = blessed.textbox({
+        parent: transferForm,
+        name: 'amount',
+        top: 9,
+        left: 0,
+        width: 20,
+        mouse: true,
+        vi: false,
+        inputOnFocus: true,
+        height: 3,
+        border: {
+            type: 'line',
+            fg: 'white'
+        },
+        fg: 'white',
+    });
+
+    amountInput.key('enter', function() {
+        transferButton.press();
+    });
+
+    amountInput.key(['escape'], function() {
+        screen.focusPop();
+        transferWindow.focus();
+    })
+
+    amountInput.key(['tab'], function() {
+        screen.focusPop();
+        transferWindow.focus();
+    })
+
+    amountInput.on('blur', function() {
+        transferWindow.focus();
+    })
+
+    // pop focus on textbox click
+    idInput.on('click', function() {
+        screen.focusPop();
+    });
+
+    let availableBalanceText = blessed.text({
+        parent: transferForm,
+        top: 9,
+        left: 21,
+        fg: 'white',
+        tags: true,
+        content: `{bold}Available:{/}\n${WB.prettyPrintAmount(walletBalanceData[0])}`
+    })
+
+    // define submit button
+    let transferButton = blessed.button({
+        parent: transferForm,
+        mouse: true,
+        shrink: true,
+        padding: {
+            left: 14,
+            right: 14
+        },
+        left: 0,
+        top: 12,
+        content: 'send (enter)',
+        style: {
+            bg: 'black',
+            fg: 'white',
+            hover: {
+                bg: 'red'
+            }
+        }
+    });
+
+    // on send transfer button press
+    transferButton.on('press', function() {
+        screen.focusPop();
+        transferForm.submit();
+        transferForm.reset();
+    });
+
+    // on transaction
+    wallet.on('transaction', async function() {
+
+        notifyUser(notificationText, 'New transation found!', 2000);
```

Comment:
```suggestion
        notifyUser(notificationText, 'New transaction found!', 2000);
```
Might also want to expand this to show amount? `New transaction found (+123 TRTL)`